lms/util/lti_launch.py

```diff
@@ -37,7 +41,10 @@ def some_view(request):
     def wrapper(request):
         """Handle the lms validation."""
         lti_params = get_lti_launch_params(request)
-        consumer_key = lti_params['oauth_consumer_key']
+        try:
+            consumer_key = lti_params['oauth_consumer_key']
+        except KeyError:
+            raise MissingLTILaunchParamError(_('Required LTI data param for lti launch missing: oauth_consumer_key'))
```

Comment:
Can we keep the error messages consistent? Previous ones seem to use the `'LTI data parameter FOO is required for launch"` format.